src/plugins/intel_cpu/src/graph_context.h

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cache/multi_cache.h"
+#include "config.h"
+#include "dnnl_scratch_pad.h"
+#include "extension_mngr.h"
+#include "weights_cache.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+class GraphContext {
+public:
+    typedef std::shared_ptr<GraphContext> Ptr;
+    typedef std::shared_ptr<const GraphContext> CPtr;
+
+    GraphContext(const Config& config,
+                 ExtensionManager::Ptr extensionManager,
+                 WeightsSharing::Ptr w_cache,
+                 std::shared_ptr<std::mutex> sharedMutex,
+                 bool isGraphQuantized)
+        : config(config),
+          extensionManager(extensionManager),
+          weightsCache(w_cache),
+          sharedMutex(sharedMutex),
+          isGraphQuantizedFlag(isGraphQuantized) {
+        rtParamsCache = std::make_shared<MultiCache>(config.rtCacheCapacity);
+        rtScratchPad = std::make_shared<DnnlScratchPad>(eng);
+    }
+
+    Config& getConfig() {
+        return config;
+    }
```

Comment:
Could you please elaborate a little bit on why we leave this ability for the config modification?

Reply:
removed